distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,105 @@
+*** CAS integration
+*** X.509 authentication
+*** Basic authentication
+*** SAML authentication
+*** Guest login support
+*** Multiple realm login support to mix and match different authentication types and services between endpoints.
+*** WS-Security, WS-SecurityPolicy, WS-Policy, WS-Trust, WS-SecureConversation, WS-Addressing
+** Provides a Security Framework (a set of APIs that define the integration with the DDF framework).
+** Provides Security Service reference implementations for a realistic end-to-end use case.
+** Role and Attribute based access control.
+** Attribute based filtering for searches performed throughout the system.
+** Federated identity through metadata catalog.
+* Search User Interface
+** 3D WebGL map built off of Cesium.
+** 2D map built off of OpenLayers 3.
+** USNG/MGRS grid support.
+** GeoNames geocoder integrated into both maps.
+** CometD integration for push notifications.
+** Upload and edit capability.
+** Saved workspaces (searches and metadata artifacts).
+* Admin Web User Interface
+** Web based install wizard.
+** Application grid to organize configurations.
+** Pluggable configuration pages for applications to simplify configurations for complex scenarios.
+** Metrics web application to view up to date system metrics.
+
+More information about {branding} is located at: https://tools.codice.org/wiki/display/{branding}[{branding} Home].
+
+=== Applications
+
+{branding} is comprised of several modular applications, to be installed or uninstalled as needed.
+
+{branding} Administrative Application::
+The administrative application enhances administrative capabilities when installing and managing {branding}. It contains various services and interfaces that allow administrators more control over their systems.
+{branding} Catalog Application::
+The {branding} Catalog provides a framework for storing, searching, processing, and transforming information.
+Clients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.
+At the core of the Catalog functionality is the *Catalog Framework*, which routes all requests and responses through the system, invoking additional processing per the system configuration.
+{branding} Content Application::
+The {branding} Content application provides a framework for storing, reading, processing, transforming and cataloging data.
+{branding} Platform Application::
+The Platform application is considered to be a core application of the distribution. The Platform application has fundamental building blocks that the distribution needs to run. +
+These building blocks include subsets of: http://karaf.apache.org/[Karaf], http://cxf.apache.org/[CXF], http://karaf.apache.org/index/subprojects/cellar.html[Cellar], and http://camel.apache.org/[Camel]. +
+Included as part of the Platform application is also a Command Scheduler.
+The Command Scheduler allows users to schedule Command Line Shell Commands to run at certain specified intervals.
+
```

Comment:
I like the empty line between these two "applications". Makes it easier to see the sections when looking at the raw asciidoc. Can we add one between all sections?